src/plugins/data_views/server/constants.ts

```diff
@@ -93,3 +93,5 @@ export const INITIAL_REST_VERSION = '2023-10-31';
  */
 
 export const INITIAL_REST_VERSION_INTERNAL = '1';
+
+export const DEFAULT_FIELD_CACHE_FRESHNESS = 5;
```

Comment:
Maybe a comment here describing the unit of `5` would be helpful (or maybe a link to some docs for this property).